docs/references/javascript/organization/membership-request.mdx

```diff
@@ -0,0 +1,33 @@
+---
+title: Organization Membership Request
+description: These are all methods on the Organization class that allow you to manage the membership requests for an organization.
+---
+
+# Organization Membership Request
+
+These are all methods on the [Organization](/docs/references/javascript/organization/organization) class that allow you to manage the membership requests for an organization.
+
+## `getMembershipRequests()`
+
+Retrieves a list of domains for the currently active organization.
+
+```typescript
+function getMembershipRequests(params?: GetMembershipRequestParams): Promise<PaginatedResources<OrganizationMembershipRequest>>;
+```
+
+### `GetMembershipRequestParams`
+
+| Name | Type | Description |
+| --- | --- | --- |
+| `initialPage?` | `number` | A number that can be used to skip the first n-1 pages. For example, if `initialPage` is set to 10, it is will skip the first 9 pages and will fetch the 10th page. |
```

Comment:
```suggestion
| `initialPage?` | `number` | The page to fetch results from. For example, if `initialPage` is set to 10, it  will skip the first 9 pages and will fetch the 10th page. |
```

Reply:
I don't know if I agree. It's not the page to fetch results from, it's the starting page to begin fetching results from.